gpii/node_modules/packagekit/nodepackagekit/nodepackagekit_test.js

```diff
@@ -155,6 +155,17 @@ var fluid = require("universal"),
         return matchPkg;
     };
 
+    pkTests.testPerformActionFailure = function() {
```

Comment:
Two linting failures in this function - please ensure to run "grunt lint" before issuing pull - see @the-t-in-rtf work on git commit hooks